### ISSUE TITLE
every buttons should have a text or ariaLable

### ISSUE THREAD

**Issue by datenangebot** [nextcloud/tables#209]
There are a lot warnings in the console: `You need to fill either the text or the ariaLabel props in the button component.`

Let's remove them.

**Comment by datenangebot** [comment]
many were fixed before, added just one more ariaLabel 3ad0fef

**Comment by arnowelzel** [comment]
This seems happen other apps as well, see https://github.com/nextcloud/text/issues/8126
